.github/workflows/ci-pullrequest-pipeline.yml

```diff
@@ -0,0 +1,29 @@
+name: Web Frontend CI Pipeline on PULL REQUEST
```

Comment:
```suggestion
name: Check Style
```

Give a meaning full name to the workflows so it is clear what they do. Same for the filename :)

Reply:
Done